pom.xml

```diff
@@ -388,6 +388,51 @@
       </dependency>
       <dependency>
         <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-common</artifactId>
```

Comment:
we still need these as they are introduced from north bound (e.g., hdfs-client, keyvalue-client) into assembly jar, even after this PR hadoop dependency is shaded from south bound